utility-emissions-channel/typescript_app/runManual.sh

```diff
@@ -6,8 +6,14 @@ rm -rf ./node_modules
 echo "Updating npm..."
 npm update
 
+echo "Setting user to root..."
+npm config set user root
```

Comment:
@Pihu1998 As a general best practice it is not recommended to run anything as root unless there's no other choice. Based on that, I'd recommend including at least a comment in the script explaining why the need for superuser privileges for the script.